tests/distributed_grids/intergrid_transfer_representation_parallel.output

```diff
@@ -0,0 +1,10 @@
+DEAL:0:2d::Checking in 2 space dimensions
```

Comment:
This test won't be run in parallel. You want to name the output file something like ```intergrid_transfer_representation_parallel.mpirun=2.output``` This will run the test with 2 processors.